docs/Introduction.md

```diff
@@ -818,7 +818,7 @@ const validate = ctx.schema('testForm');
 
 ### Home
 
-The directory in which the main application script resides, usually called `index.js` is considered the application
+The directory in which the main application script, usually called `index.js`, resides, is considered the application
```

Comment:
This looks weird to me.

Reply:
At a minimum I think there should be a comma after `index.js` in the original version.
Feel free to remove or change this, I have allowed edits by maintainers.

Reply:
Yes, just add the comma after `index.js`, and leave the sentence as is otherwise.

Reply:
Done.

Reply:
How about:

The directory in which the main application script (usually called `index.js`) resides is considered the application home directory.

Reply:
That would have worked too.